elated-egrets/bangazon-internal-site
# J remployee list

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRemployee_list
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. run tests to make sure they pass
1. check server to make sure there are no errors

## What you changed
added the testing for the employee list
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
